src/snowflake/connector/auth/_auth.py

```diff
@@ -239,20 +246,12 @@ def authenticate(
             {k: v for (k, v) in body["data"].items() if k != "PASSWORD"},
         )
 
-        # accommodate any authenticator specific timeout requirements here.
-        # login_timeout comes from user configuration.
-        # Between login timeout and auth specific
-        # timeout use whichever value is smaller
-        auth_timeout = min(self._rest._connection.login_timeout, auth_instance.timeout)
-        logger.debug(f"Timeout set to {auth_timeout}")
```

Comment:
we can add back the debug log for `timeout`, in the code change which should be `auth_instance.socket_timeout_override` I think?